addons/crm/tests/test_crm_lead_assignment.py

```diff
@@ -461,6 +461,63 @@ def test_assign_preferred_domain(self):
         self.assertMemberAssign(test_sales_team_m2, 3)
         self.assertMemberAssign(test_sales_team_m3, 3)
 
+    def test_assign_preferred_and_probability(self):
+        random.seed(1914)
+        preferred_tag = self.env['crm.tag'].create({'name': 'preferred'})
+        leads = self._create_leads_batch(
+            lead_type='lead',
+            user_ids=[False],
+            count=10,
+        )
+        proba = 0
+        for lead in leads[:5]:
+            lead.write({
+                'tag_ids': [(6, 0, preferred_tag.ids)],
+                'probability': proba,
+            })
+            proba += 1
```

Comment:
Runbot seems not happy with the style
```suggestion
        for proba, lead in enumerate(leads[:5]):
            lead.write({
                'tag_ids': [(6, 0, preferred_tag.ids)],
                'probability': proba,
            })
```